junit-launcher/src/main/java/org/junit/gen5/launcher/main/DefaultLaunchParameter.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2015-2016 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit.gen5.launcher.main;
+
+import static org.junit.gen5.commons.meta.API.Usage.Experimental;
+
+import java.util.Objects;
+
+import org.junit.gen5.commons.meta.API;
+import org.junit.gen5.commons.util.ToStringBuilder;
+import org.junit.gen5.launcher.LaunchParameter;
+
+/**
+ * @since 5.0
+ */
+@API(Experimental)
+public class DefaultLaunchParameter implements LaunchParameter {
+	private static final long serialVersionUID = 1L;
+
+	public static LaunchParameter keyValuePair(String key, String value) {
```

Comment:
Actually, I think we can get rid of the interface and just have this class. Moreover, since `LaunchParameter` is only used in the builder we can just use strings?